tests/acceptance/program/overview-test.js

```diff
@@ -102,7 +102,7 @@ test('change duration', function(assert) {
 test('leave duration at 1', function(assert) {
   server.create('program', {
     school: 1,
-    duration: null,
+    duration: 1
```

Comment:
Thats ok since a value is required here you should also change the program creation to set a default of 1 for duration.  It looks like it was setup to not require a value, but that is wrong.